api/capi/src/nnstreamer-capi-single.c

```diff
@@ -431,6 +443,9 @@ ml_single_close (ml_single_h single)
   }
 
   single_h = (ml_single *) single;
+  ML_SINGLE_MAGIC_CHECK (single_h);
```

Comment:
Discussion about validating the handle (maybe TODO list)
How about checking magic inside the mutex lock?
IMO, single_h->is_valid is unnecessary with single-magic.

Reply:
`is_valid` has been removed
I think checking inside lock will assume that the lock is valid. However, if the object is invalid, then the lock can be invalid as well, which can cause the program to crash when trying to acquire the lock. Please correct me if I am wrong.